src/main/resources/paas.yaml

```diff
@@ -26,8 +26,8 @@ externalConfigDirectory: /tmp
 
 downloadConfigs: true
 
-fieldsYamlLocation: classpath://config/fields.yaml
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: https://field.discovery.openregister.org/records.yaml
```

Comment:
I was thinking more along the lines that we allow the phase to be specified (via register domain right?), so this probably shouldn't be pointing directly at discovery.